Formula/vgmstream.rb

```diff
@@ -15,21 +14,21 @@ class Vgmstream < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "ffmpeg"
   depends_on "libao"
   depends_on "libvorbis"
   depends_on "mpg123"
-  depends_on "ffmpeg"
 
   def install
     system "cmake", "-DBUILD_AUDACIOUS:BOOL=OFF", "."
     system "make", "vgmstream_cli"
     system "make", "vgmstream123"
-    bin.install "cli/vgmstream-cli"
+    bin.install "cli/vgmstream_cli"
```

Comment:
Will it be surprising to users if the binary is renamed from `vgmstream-cli` to `vgmstream_cli`?

Reply:
The purpose of adding `cmake` command is to enable FFmpeg to supports more codecs but the binary name will change. The document of vgmstream mentioned another build method(https://github.com/losnoco/vgmstream/blob/master/doc/BUILD.md#audacious-plugin) doesn't rename the binary but I don't know how to use it correctly(disable audacious and enable ffmpeg).

Reply:
you should make a symlink `bin.install_symlink "vgmstream_cli" => "vgmstream-cli"`

Reply:
Thanks!